app/invocation/invocation_model.tsx

```diff
@@ -257,7 +257,14 @@ export default class InvocationModel {
     return this.started?.command || "build";
   }
 
+  getToolName(): ToolName {
+    return this.invocations.find(() => true).role === "CI_RUNNER" ? "ci_runner" : "bazel";
+  }
+
   getTool() {
+    if (this.getToolName() === "ci_runner") {
+      return "ci_runner";
```

Comment:
Do we want to expose the string `ci_runner` to the user?

Reply:
I was thinking this is what we'd show in the "details" page -- I can delete this if you think we should just omit that row instead.

Reply:
Was just thinking we could show a more user friendly string like `BuildBuddy Workflow` or something like that

Reply:
Went ahead and deleted this.